src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,21 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+#### Asking complex questions without using hint text
+
+Do not use hint text if you need to give a lengthy explanation with lists and paragraphs. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+You should also avoid using links in hint text, as screen readers might not read out them out.
+
+If you're asking a question that needs a detailed explanation, use:
+* a `h1` heading that's a statement (for example, 'Add undergraduate degree') rather than a question
+* whatever mix of text, paragraphs, lists and examples best explains your question to users
+* a label, above the form input, that asks users a specific question – for example, 'What type of degree do you want to add?'
```

Comment:
I'm struggling with this a bit, as the bullet points could be ordered in 2 different ways — listing this pattern top-to-bottom as you've done, or by how someone would rephrase a question page to this new pattern.

Tried writing something that does a bit of both?

```suggestion
If you're asking a question that needs a detailed explanation, introduce this information first and use:
* the `h1` heading to describe the overall task (for example, 'Add undergraduate degree') instead of asking the question
* whatever mix of text, paragraphs, lists and examples best explains your question to users
* the label just above the form input to ask the question (for example, 'What type of degree do you want to add?')
```